### ISSUE TITLE
Support for default Value

### ISSUE THREAD

**Issue by notz** [dozd/mongo-mapper#12]
Very helpful for updates, if a document has a new int field with default value x. 
Currently you have to update all document in an manual process before release, otherwise the documents can't be read.

**Comment by dozd** [comment]
Or you can use `Integer`?

**Comment by notz** [comment]
Yes, but then i always need a null check in the application which is not really nice. I would prefer a default value and no null check. Or at least don't throw a exception on this case -> the object already has a default value.

**Comment by dozd** [comment]
Ok I see the point. It should not be large change.
